README.md

```diff
@@ -77,14 +77,19 @@ In the `./nodezoo-system` folder run,
 fuge build fuge/system.yml
 ```
 
-### 6. Start the infrastructure
+### 6. Insert GitHub token
+For the GitHub service to work you must insert your GitHub personal access token in to the sample.env file located in fuge/env.
```

Comment:
Would we not just use system.env instead of needing to copy everything over? @mcdonnelldean